rotkehlchen/chain/bitcoin/__init__.py

```diff
@@ -13,65 +15,130 @@ def _have_bc1_accounts(accounts: List[BTCAddress]) -> bool:
     return any(account.lower()[0:3] == 'bc1' for account in accounts)
 
 
-def get_bitcoin_addresses_balances(accounts: List[BTCAddress]) -> Dict[BTCAddress, FVal]:
-    """Queries blockchain.info or blockstream for the balances of accounts
+def _query_blockstream_info(accounts: List[BTCAddress]) -> Dict[BTCAddress, FVal]:
```

Comment:
This and `_query_mempool_space` are identical with the only difference being the url. It makes no sense to have them as separate functions. Please combine them into one and have the `url`as a function parameter.